cmd/notifier/config.go

```diff
@@ -36,8 +36,8 @@ type notifierConfig struct {
 	Timezone string `yaml:"timezone"`
 	// Format for email sender. Default is "15:04 02.01.2006". See https://golang.org/pkg/time/#Time.Format for more details about golang time formatting.
 	DateTimeFormat string `yaml:"date_time_format"`
-	// Image store config section.
-	ImageStore map[string]string `yaml:"image_store"`
+	// Image stores config section.
+	ImageStores map[string]map[string]string `yaml:"image_store"`
```

Comment:
Let's make more stupid solution here. Just create some structures:
```
type imageStoreConfig struct {
	S3 s3Config `yaml:"s3"`
}

type s3Config struct {
	AccessKeyID string `yaml:"access_key_id"`
	AccessKey   string `yaml:"access_key"`
	Region      string `yaml:"region"`
	Bucket      string `yaml:"bucket"`
}
```

And replace `map[string]map[string]string` to `imageStoreConfig.
I do not see much point in using here such a flexible structure as `map[string]map[string]string` because you are hardcoded store type in `image_store.go` at the same time, it greatly complicates the code itself and its reading.

Reply:
You can also add `enabled` field for an accurate transmission of whether the image store is enabled or not, or you can simply transfer the already written parameters: if configured, the image store is enabled, if not, then disabled.